packages/dashboard/src/store/actions/createWidget/presets/index.ts

```diff
@@ -30,6 +31,8 @@ export const widgetCreator =
       return appKitWidgetCreator(componentTag as AppKitComponentTag, preset);
     } else if (componentTag === 'text') {
       return textWidgetCreator(componentTag, preset);
+    } else if (componentTag === 'input') {
```

Comment:
I'm worried this part is not scalable. Noticed we have conditions for `text widget` in multiple places and I just followed this pattern for now. I want to tackle this at some point.